lib/rspec/core/formatters/html_formatter.rb

```diff
@@ -69,10 +69,11 @@ def example_failed(failure)
           example = failure.example
 
           exception = failure.exception
+          message_lines = failure.fully_formatted_lines(nil, RSpec::Core::Notifications::NullColorizer)
           exception_details = if exception
                                 {
-                                  :message => failure.message_lines.join("\n"),
-                                  :backtrace => failure.formatted_backtrace.join("\n")
+                                  :message => message_lines.drop(2).join("\n"),
```

Comment:
Not a merge blocker, but I think our future selves will be confused about what lines are dropped by `drop(2)`.  Can you leave a comment mentioning what lines this is intended to drop?